src/main/resources/db/migration/V0003__Create_table_animal_breeds.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE `animal_breeds` (
```

Comment:
Why we divided sql queries to more one file ?

Reply:
I guess that for migration and develop DB must be more than  one file.Step by stap. If I am wrong, I will make one file.